scripts/start-rc.bat

```diff
@@ -78,6 +78,9 @@ if exist "hazelcast-enterprise-%HAZELCAST_ENTERPRISE_VERSION%-tests.jar" (
 )
 
 
+REM necessary arguments for Java 9+
```

Comment:
why? Can you point to some link explaining this pls?

Reply:
> why? Can you point to some link explaining this pls?

We do this in _lots_ of places and don't document it anywhere, including the Linux flavour of this script:
https://github.com/hazelcast/hazelcast-cpp-client/blob/a79e606c0ec186a1aa86430942fc3f95ab1b30c8/scripts/start-rc.sh#L121

The short answer is [Project Jigsaw](https://www.baeldung.com/project-jigsaw-java-modularity).